# 2022-03-04 18:47:35.903049: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE3 SSE4.1 SSE4.2 AVX AVX2 AVX512F FMATo enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.2022-03-04 18:47:35.905247: W tensorflow/core/framework/op_kernel.cc:1692] OP_REQUIRES failed at spacetobatch_op.cc:217 : Invalid argument: block_shape rank should be 1 instead of 0Traceback (most recent call last):  File "/media/ivysyn/tensorflow/synthesized/all-crashes/all/SpaceToBatchND_d0c13c1f5a4cf9f52134d2b37dada7a4.py", line 8, in <module>    tf.raw_ops.SpaceToBatchND(input=input, block_shape=block_shape, paddings=paddings)  File "/media/anaconda3/envs/ivysyn/lib/python3.9/site-packages/tensorflow/python/util/tf_export.py", line 404, in wrapper    return f(**kwargs)  File "/media/anaconda3/envs/ivysyn/lib/python3.9/site-packages/tensorflow/python/ops/gen_array_ops.py", line 9799, in space_to_batch_nd    _ops.raise_from_not_ok_status(e, name)  File "/media/anaconda3/envs/ivysyn/lib/python3.9/site-packages/tensorflow/python/framework/ops.py", line 6941, in raise_from_not_ok_status    six.raise_from(core._status_to_exception(e.code, message), None)  File "<string>", line 3, in raise_fromtensorflow.python.framework.errors_impl.InvalidArgumentError: block_shape rank should be 1 instead of 0 [Op:SpaceToBatchND]

# SpaceToBatchNDOp

import tensorflow as tf

input = tf.constant(-3.5e+35, shape=[10,19,22], dtype=tf.float32)
block_shape = tf.constant(65534, shape=[], dtype=tf.int32)
paddings = tf.constant(0, shape=[2,2], dtype=tf.int32)
tf.raw_ops.SpaceToBatchND(input=input, block_shape=block_shape, paddings=paddings)